introduction_and_learning/Histogram/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
.idea files should be added to .gitignore as all of them are from internal Pycharm settings